src/connect.ts

```diff
@@ -47,16 +46,29 @@ export async function connect(config: ConnectConfig): Promise<Near> {
                 // TODO: Only load key if network ID matches
                 const keyPair = accountKeyFile[1];
                 const keyPathStore = new InMemoryKeyStore();
-                await keyPathStore.setKey(config.networkId, accountKeyFile[0], keyPair);
+                await keyPathStore.setKey(
```

Comment:
nit: feels like this and following reformatting isn't related to content of PR (generally best avoid making unnecessary changes to not mess with `git blame`)  + seems to use vertical space a bit too agressively